aqua/resources-api.aqua

```diff
@@ -309,22 +303,21 @@ func resolveProviders(resource_id: ResourceId, ack: i16) -> []Record, *Error:
     nodes <- getNeighbours(resource_id)
     res: *[]Record
     error: *Error
+    successful: *bool
```

Comment:
Why do you need this? You could join on `res`, no?

Reply:
if so i need to define another `wait` func just with another type for first arg